### PR TITLE
#574294: Automatically run tslint after CI test

### PR DESCRIPTION
**Pull request by reinouts**
...and solve all tslint warnings in one go.

### CODE REVIEW

**Comment by ismailhabib** [comment]
:+1:

File: mendix-platform-sdk.ts [hunk 1]
```diff
@@ -351,9 +351,9 @@ export class PlatformSdkClient {
 
 		return {
 			path: PlatformSdkClient.PROJECTS_API_PATH,
-			method: 'POST',
+			method: "POST",
 			headers: {
-				'Content-Type': 'text/xml;charset=UTF-8'
+				"Content-Type": "text/xml;charset=UTF-8"
```
Comment (dlhartveld) [inline mendix-platform-sdk.ts]: Is there a reason why you do not use `backticks` here?

Comment (reinouts) [inline mendix-platform-sdk.ts]: Well spotted, but yes there is a reason. TSC fails to compile JSON objects with `backtick` strings.

Comment (dlhartveld) [inline mendix-platform-sdk.ts]: For both the property name and value?

Comment (reinouts) [inline mendix-platform-sdk.ts]: Good point, it's just for the name. Matter of taste, but I'd prefer the same quote style for name and value unless we really need to template the value.

**Pull request merged**